### ISSUE TITLE
shuffle parameter ignored in fit_generator with num_workers=0

### ISSUE THREAD

**Issue by alexeyr** [keras-team/keras#12927]
**System information**  
- Have I written custom code (as opposed to using example directory):  no
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):  Linux Ubuntu 16.04
- TensorFlow backend (yes / no):  yes
- TensorFlow version:  0.13.4
- Keras version:  2.2.4
- Python version:  3.5.2
- CUDA/cuDNN version:  n/a
- GPU model and memory:  n/a

**Describe the current behavior**  
In https://github.com/keras-team/keras/blob/master/keras/engine/training_generator.py#L153-L169, when `num_workers` is 0, and the generator is a `Sequence`, the `shuffle` parameter is ignored.

**Describe the expected behavior**  
If `shuffle` is `True`, the sequence should be iterated in a shuffled order. 

I'd be happy to contribute a fix if necessary.

**Comment by jvishnuvardhan** [comment]
I can see `tensorflow.keras` has similar code starting at this [line](https://github.com/tensorflow/tensorflow/blob/0306c3cb4a6710d61c5f90bc6b01e32426e9566f/tensorflow/python/keras/engine/training_generator.py#L510). Thanks!
